indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -33,57 +24,55 @@ def tconf(tconf):
 def client_send_revoc_reg_def(looper,
```

Comment:
According to what the method does, it needs to be called trust_anchor_sends_revoc_reg_def`